contentcuration/contentcuration/utils/assessment/base.py

```
@@ -0,0 +1,383 @@
+            f.write(content)
+        self.files_to_write.append(full_path)
+
+    def _add_original_image(self, checksum, filename, new_file_path):
+        """Extract original image handling"""
+        with storage.open(
+            models.generate_object_storage_name(checksum, filename), "rb"
+        ) as imgfile:
+            original_content = imgfile.read()
+        self.add_file_to_write(os.path.join(new_file_path, filename), original_content)
+
+    def _get_similar_image(self, filename, width, height):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)